### DISCUSSION TITLE
https://angular.love/pl/angular-19-co-nowego

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#354] [General]
# https://angular.love/pl/angular-19-co-nowego

Angular.love - ciekawostki oraz rozwiązania dla średnio-zaawansowanych oraz zaawansowanych developerów Angulara.

https://angular.love/pl/angular-19-co-nowego

**Comment by 0xFACE** [comment]
Przydałaby się migracja atrybutów [color]  po zmianie Material2 na Material3.

**Comment by sebmat003** [comment]
equals: arraysAreEqual, // Niestandardowa funkcja porównująca tablice


Powinno być chyba equal, a nie equals.

https://github.com/angular/angular/blob/main/packages/core/rxjs-interop/src/to_signal.ts#L30-L81

**Reply by sebmat003** [reply]
Btw, bardzo fajny artykuł :)
